Functions/BIDS/xASL_bids_BIDSifyASLJSON.m

```diff
@@ -64,12 +64,21 @@
 jsonInMerged = xASL_bids_MergeStudyPar(jsonIn, studyPar, 'asl');
 	
 %% 3. Extract the scaling factors from the JSON header
+% Fujifilm fix
+if isfield(jsonInMerged, 'StationName') && ~isfield(jsonInMerged, 'Manufacturer')
```

Comment:
Fine. But not very transparent to add unrelated changes to this issue. This should have been done and tested in a Fujifilm issue. Next time ;)

Reply:
You're right. It was just a very small fix, and you never know this is really Fujifilm-specific. Manufacturer could always be missing (we could issue a warning though).